internal/testing/octest/exporter.go

```diff
@@ -38,7 +40,11 @@ func NewTestExporter(views []*view.View) *TestExporter {
 
 	// Register for metrics.
 	view.RegisterExporter(te)
-	view.SetReportingPeriod(time.Millisecond)
+	// The reporting period will affect how long it takes to get stats (view.Data).
+	// We want it short so tests don't take too long, but long enough so that all
```

Comment:
This looks flaky so it makes me a bit nervous. Can you clarify "long enough so that all the work gets done"? It looks like `Counts` waits until data arrives...?

Reply:
I'm not fond of it either. I attempted to clarify. PTAL.

Reply:
I'll approve for now but it might be worth thinking about how to make this less racy. For example, can `diffCounts` retry until it matches expectation (with a relatively long cap like 10s so actually failing tests don't hang forever)?